linera-views/src/indexed_db.rs

```diff
@@ -62,6 +71,12 @@ impl IndexedDbStore {
         let object_store = transaction.object_store(&self.object_store_name)?;
         Ok(f(object_store))
     }
+
+    fn extend_key(&self, key: &[u8]) -> Vec<u8> {
```

Comment:
Maybe `full_key`?

(Not sure what the best nomenclature is: "full key" vs "subkey"? "Rooted key" vs. "rootless key"? "Long key" vs. "short key"?)